catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -150,10 +150,13 @@ public synchronized void failSource(String sourceId) {
         failedResponse.getProperties()
                 .put("elapsed", -1L);
 
-        updateStatus(sourceId, failedResponse);
+        updateStatus(sourceId, failedResponse, cause);
     }
 
-    private void updateStatus(String sourceId, QueryResponse queryResponse) {
+    /**
+     * Note that {@param cause} could be null if no {@link Exception} was thrown.
+     */
+    private void updateStatus(String sourceId, QueryResponse queryResponse, Exception cause) {
```

Comment:
The `QueryResponse` processing details include a list of exceptions returned for failed sources.  Those exceptions should be included in the reasons.